### PR TITLE
Feat: Support custom deploy branch

### PR DESCRIPTION
**Pull request by neilkuan**
Feat: Support custom deploy branch

### CODE REVIEW

**Comment by neilkuan** [comment]
Hi @victoriadrake , ready to review. 😊 😊 😊

File: action.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ author: 'victoriadrake'
 branding:
   icon: 'send'
   color: 'blue'
+inputs:
+  branch:
+    description: "which branch of remote repo will be deploy. Default: master"
```
Comment (victoriadrake) [inline action.yml]:
```suggestion
    description: "The branch of your remote repository to deploy to. Defaults to 'master'"
```

File: README.md [hunk 2]
```diff
@@ -68,6 +68,38 @@ jobs:
         uses: victoriadrake/hugo-remote@master
 ```
 
+Here is an example workflow file that uses this action on any `push` event to the `main` branch:
```
Comment (victoriadrake) [inline README.md]:
```suggestion
You can customize branch names in your configuration. Here is an example workflow file that uses this action on any `push` event to the `master` branch and deploys to the remote repository's `main` branch:
```

**Review by victoriadrake — APPROVED** [review]
Thank you for the PR Neil!

**Pull request merged**